src/Workspaces/Core/Portable/Storage/StorageOptions.cs

```diff
@@ -17,7 +17,9 @@ internal static class StorageOptions
 
         public const string OptionName = "FeatureManager/Storage";
 
-        public static readonly Option<StorageDatabase> Database = new(OptionName, nameof(Database), defaultValue: StorageDatabase.SQLite);
+        public static readonly Option<StorageDatabase> Database = new(
+            OptionName, nameof(Database), defaultValue: StorageDatabase.SQLite,
+            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(Database)));
```

Comment:
❔ Do we have other cases where an enum is stored directly? I'm assuming we do and this works but maybe a quick check to make sure?